Bio/Align/__init__.py

```diff
@@ -1068,6 +1071,193 @@ def parse_printed_alignment(cls, lines):
         parser.fill(coordinates)
         return sequences, coordinates
 
+    @classmethod
+    def from_alignments_with_same_reference(
+        cls, pwas: list["Alignment"] | tuple["Alignment"]
```

Comment:
Can you rename `pwas` to `alignments` or something similar? `pwas` suggests pairwise alignments. Currently this method supports pairwise alignments only, but it would be straightforward to extend this to multiple alignments.